cms/test_utils/project/placeholderapp/views.py

```diff
@@ -14,8 +15,8 @@ def example_view(request):
     return render(request, 'placeholderapp.html', context)
 
 
-def _base_detail(request, instance, template_name='detail.html',
-                 item_name="char_1", template_string='',):
+def _base_detail(request, instance, template_name='detail.html', item_name="char_1",
```

Comment:
As a purist, I would like if you tried to avoid making unrelated code cleanups like this.